src/huggingface_hub/repocard.py

```diff
@@ -544,6 +544,7 @@ def metadata_eval_result(
     dataset_id: str,
     metrics_config: Optional[str] = None,
     metrics_verified: bool = False,
+    metrics_token: Optional[str] = None,
```

Comment:
This might be a breaking change if users are using positional args in their code. Happy to move this to the end of the signature if needed

Reply:
Thanks, done in [0e1b34d](https://github.com/huggingface/huggingface_hub/pull/1142/commits/0e1b34dc1d315a3da6d5c19bfef7238842888f55)